wit/core.wit

```diff
@@ -0,0 +1,89 @@
+package wasmcloud:core@0.1.0
+
+interface types {
+  type link-settings =          tuple<string, string>
+  type host-env-values =        tuple<string, string>
+  type actor-links =            list<link-defination>
+  type cluster-issuers =        list<cluster-issuer-key>
+  type cluster-issuer-key =     string 
+  type capability-contract-id = string
+  type blob =                   list<u8>
+  type trace-context =          tuple<string, string>
```

Comment:
like a couple of the other ones, this probably needs to be `list<tuple<string, string>>`